### PR TITLE
Retargets link at /advanced. Closes #677

### LINKED ISSUE #677: Change search "start over" link
When starting new search users expect "start over" to bring them to new search (blank search box), not a list of all search results.  

screen shot 2015-10-05 at 5 51 30 pm

### PR DESCRIPTION
**Pull request by mbarnett**

### CODE REVIEW

File: app/views/catalog/_constraints.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
       <div id="appliedParams" class="clearfix constraints-container">
         <div class="pull-right">
-          <%=link_to t('blacklight.search.start_over'), start_over_path, :class => "catalog_startOverLink btn btn-sm btn-text", :id=>"startOverLink" %>
+          <%=link_to t('blacklight.search.start_over'), advanced_path, :class => "catalog_startOverLink btn btn-sm btn-text", :id=>"startOverLink" %>
```
Comment (murny) [inline app/views/catalog/_constraints.html.erb]: camelcase/underscore for css class/id naming ewww haha.

File: spec/features/catalog_search_spec.rb [hunk 2]
```diff
@@ -105,4 +105,11 @@
     expect(page).to_not have_content(@gfS.title.first)
   end
 
+  it 'should have a start over link that takes you to /advanced' do
```
Comment (murny) [inline spec/features/catalog_search_spec.rb]: Nice one for adding a test! :+1:

**Comment by murny** [comment]
Looks good to me! :shipit:

File: app/views/catalog/_constraints.html.erb (hunk 1 above)
Comment (mbarnett) [inline app/views/catalog/_constraints.html.erb]: Yeah, [Blacklight's CSS is full of it](https://github.com/projectblacklight/blacklight/blob/6e9151de47e74a1dc8efea7d2e87effa02b8a816/app/assets/stylesheets/blacklight/_catalog.css.scss). Some kind of weird SMACSS-style semantic thing I assume

**Pull request merged**